packages/opentelemetry-types/src/trace/tracer_registry.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Tracer } from './tracer';
+
+/**
+ * TracerRegistry provides an interface for creating {@link Tracer}s
+ */
+export interface TracerRegistry {
+  /**
+   * Returns a Tracer, creating one if one with the given name and version is not already created
+   *
+   * If there is no Span associated with the current context, null is returned.
+   *
+   * @returns Tracer A Tracer with the given name and version
+   */
+  getTracer(name?: string, version?: string): Tracer;
```

Comment:
Based off the comments [here](https://github.com/open-telemetry/opentelemetry-specification/pull/354/files#diff-ea4f4a46fe8755cf03f9fb3a6434cb0cR98), I imagine name will go back to being required

Reply:
That seems to be the way the wind is blowing